pkg/test-infra/tidb/recommend.go

```diff
@@ -220,11 +249,23 @@ func RecommendedTiDBCluster(ns, name string, clusterConfig fixture.TiDBClusterCo
 					"app.kubernetes.io/component": "tidb",
 					"app.kubernetes.io/name":      "tidb-cluster",
 				},
-				ClusterIP: "",
 			},
 		},
 	}
 
+	if clusterConfig.TiFlashReplicas > 0 {
+		r.EnableTiFlash(clusterConfig.TiFlashImageVersion, clusterConfig.TiFlashReplicas)
+
+		// TODO(yeya24): There is a bug in the operator side so we can only set it
+		// manually https://github.com/pingcap/tidb-operator/issues/2219.
+		// Remove this after it fixes the bug.
```

Comment:
So it's ok to use tidb-operator with TiFlash now? I see DanielZhangQD said that it's still in progress?

Reply:
For deploying tiflash it is okay now. WIP is meant for scale in and scale out, which is not necessary in tipocket